src/main/java/edu/wpi/grip/ui/ExceptionView.java

```diff
@@ -0,0 +1,191 @@
+        messageContent.add(inputBox, 0, 1);
+        this.getDialogPane().setContent(messageContent);
+
+        final Label issuePasteLabel = new Label("Please paste this into an issue on the project GitHub:");
+
+        issueText.setEditable(false);
+        issueText.setWrapText(true);
+
+        issueText.setMaxWidth(Double.MAX_VALUE);
+        issueText.setMaxHeight(Double.MAX_VALUE);
+        GridPane.setVgrow(issueText, Priority.ALWAYS);
+        GridPane.setHgrow(issueText, Priority.ALWAYS);
+
+        final GridPane expContent = new GridPane();
+        expContent.setMaxWidth(Double.MAX_VALUE);
+        expContent.add(issuePasteLabel, 0, 0);
+        expContent.add(issueText, 0, 1);
+
+        // Set expandable Exception into the dialog pane.
+        this.getDialogPane().setExpandableContent(expContent);
+        this.getDialogPane().setExpanded(true);
+
+
+        // Prevent these two buttons from causing the alert to close
+        final Button copyToClipboardBtn = (Button) this.getDialogPane().lookupButton(copyToClipboardBtnType);
+        copyToClipboardBtn.addEventFilter(ActionEvent.ACTION, event -> {
+            final ClipboardContent content = new ClipboardContent();
+            content.putString(issueText.getText());
+            Clipboard.getSystemClipboard().setContent(content);
+            // Prevent the dialog from closing
+            event.consume();
+        });
+
+        final Button openGitHubIssueBtn = (Button) this.getDialogPane().lookupButton(openGitHubIssuesBtnType);
+        openGitHubIssueBtn.addEventFilter(ActionEvent.ACTION, event -> {
+            services.showDocument(PROJECT_ISSUE_LINK);
+            // Prevent the dialog from closing
+            event.consume();
+        });
+
+        // Set the initial focus to the input box so the cursor goes there first
+        Platform.runLater(() -> inputBox.requestFocus());
+    }
+
+    /**
+     * Iterates through the causes until the root cause is found.
+     * @param throwable The throwable to iterate through.
+     * @return The initial throwable
+     */
+    private Throwable generateInitialCause(Throwable throwable){
+        if(throwable.getCause() == null){
+            return throwable;
+        } else {
+            return generateInitialCause(throwable.getCause());
+        }
+    }
+
+    /**
+     * Takes the throwable and generates the markdown for the exception.
+     * @param throwable The initial throwable passed to the constructor
+     * @return The markdown for the exception.
+     */
+    private String generateExceptionMessage(Throwable throwable){
+        final StringWriter sw = new StringWriter();
+        final PrintWriter pw = new PrintWriter(sw);
+        throwable.printStackTrace(pw);
+
+        return new StringBuilder(sw.toString()
+                /* Allow users to maintain anonymity */
+                .replace(System.getProperty("user.home"), "$HOME").replace(System.getProperty("user.name"),"$USER"))
```

Comment:
I don't think this is really necessary.  If someone contacts us with an issue, they must have a github account or an email, so we would know who they are.  If they use only psuedonyms on github, they can probably remove their username manually.